ipfs-10-to-11/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/ipfs/fs-repo-migrations/ipfs-10-to-11
+
+go 1.14
+
+require (
+	github.com/ipfs/fs-repo-migrations v1.6.4
+	github.com/ipfs/go-blockservice v0.1.2
+	github.com/ipfs/go-cid v0.0.7 // indirect
+	github.com/ipfs/go-datastore v0.4.5
+	github.com/ipfs/go-ds-flatfs v0.4.5
+	github.com/ipfs/go-ds-leveldb v0.4.2
+	github.com/ipfs/go-filestore v1.0.0
+	github.com/ipfs/go-ipfs-blockstore v1.0.0
```

Comment:
should use the pre v1 filestore + blockstore (same versions as in go-ipfs).

Reply:
Done in [795e3b0](https://github.com/ipfs/fs-repo-migrations/pull/112/commits/795e3b0f85f0e01e10662ff64a050b533dd21724)